account_statement_import_camt54/models/account_statement_import.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+import logging
+
+from odoo import models
+
+_logger = logging.getLogger(__name__)
+
+
+class AccountStatementImport(models.TransientModel):
+    """Add process_camt method to account.bank.statement.import."""
+
+    _inherit = "account.statement.import"
+
+    # pylint: disable=W8110
```

Comment:
Question, should be favor a more human-readable pylint disable ?

Reply:
I'll add a comment or try again to fix pylint issue, thx